src/client/components/map/MapToken.tsx

```diff
@@ -239,8 +244,8 @@ function MapTokenInner({
       ref={ref}
       angle={object.rotation}
       pivot={{ x: tokenSize / 2, y: tokenSize / 2 }}
-      x={position.x + tokenSize / 2}
-      y={position.y + tokenSize / 2}
+      x={position.x + tokenSize / 2 + (isGhost ? getSmallTokenOffset() : 0)}
```

Comment:
Is there a smarter way to do this, so that I don't have to apply the offset in two places? When I moved the token correctly the health bars were suddenly off...

Reply:
I've looked into this for a bit, and I think the best way to do it is to move the second addition of the offset outside of this function to where the ghost is actually rendered. I pushed a commit that does that, and also fixes rendering of auras for small tokens.
I still don't really like the complexity of the MapToken component, but it looks a bit better to me now.

Reply:
Oh yeah, good call! I didn't even think of the auras...
I guess the complexity of the component is a problem for future us :D